pxd.c

```diff
@@ -896,6 +900,9 @@ ssize_t pxd_add(struct fuse_conn *fc, struct pxd_add_ext_out *add)
 	struct pxd_device *pxd_dev_itr;
 	int new_minor;
 	int err;
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5,15,0)
+	int irc;
```

Comment:
whats the purpose of this defining this local variable, if this is not checked anywhere.

Reply:
Thanks for looking LNS,  Sorry I should have added a comment.  This actually came up when I was checking these changes against other kernel builds so it failed on "5.16.5-1.el8.elrepo.x86_64"  kernel.  The declaration for `device_add_disk` changed in 5.16 to __must_check...   If you know a better way to handle this please let me know?
 I tried "(void)device_add_disk" and that did not work
KERNELPATH=/usr/src/kernels/5.16.5-1.el8.elrepo.x86_64 make  clean all
...
/usr/src/px-fuse-rc7/pxd.c: In function ‘pxd_add’:
/usr/src/px-fuse-rc7/pxd.c:973:9: error: ignoring return value of ‘device_add_disk’ declared with attribute ‘warn_unused_result’ [-Werror=unused-result]
  973 |         device_add_disk(&pxd_dev->dev, pxd_dev->disk, NULL);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors